package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "optimizer",
+  "version": "0.0.1",
+  "description": "This library will optimize the AsyncAPI specification file.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc"
```

You should also add scripts related to automatic workflows:

```suggestion
    "test": "echo \"No test specified\"",
    "build": "tsc",
    "lint": "echo \"Skip linting\"",
    "get:version": "echo $npm_package_version",   
    "get:name": "echo $npm_package_name",
    "generate:assets": "echo \"Skip generation\"",
    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
```

We need them for workflows https://github.com/asyncapi/.github/blob/master/.github/workflows/if-nodejs-pr-testing.yml and https://github.com/asyncapi/.github/blob/master/.github/workflows/if-nodejs-version-bump.yml

And of course we shouldn't return the `0` flag from not implemented script :)